keyboards/dztech/dz60rgb/keymaps/mekanist/keymap.c

```diff
@@ -8,38 +8,38 @@
 #define _LAYER6 6
 #define _LAYER7 7
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-	[_LAYER0] = LAYOUT( /* Base */
-		KC_GESC,  KC_1,     KC_2,     KC_3,  KC_4,  KC_5,  KC_6,    KC_7,  KC_8,    KC_9,     KC_0,            KC_MINS,  KC_EQL,  KC_BSPC, \
-		KC_TAB,   KC_Q,     KC_W,     KC_E,  KC_R,  KC_T,  KC_Y,    KC_U,  KC_I,    KC_O,     KC_P,            KC_LBRC,  KC_RBRC, KC_BSLASH, \
+	[_LAYER0] = LAYOUT( /* MAC */
+		KC_GESC,  KC_1, 	KC_2, 	  KC_3,  KC_4,  KC_5,  KC_6,	KC_7,  KC_8,	KC_9,	  KC_0,			   KC_MINS,  KC_EQL,  KC_BSPC, \
+		KC_TAB,   KC_Q,     KC_W,     KC_E,  KC_R,  KC_T,  KC_Y,    KC_U,  KC_I,    KC_O,     KC_P,			   KC_LBRC,  KC_RBRC, KC_BSLS, \
 		CTL_T(KC_CAPS),  KC_A,     KC_S,     KC_D,  KC_F,  KC_G,  KC_H,    KC_J,  KC_K,    KC_L,     KC_SCLN,         KC_QUOT,           KC_ENT, \
 		KC_LSFT,  KC_Z,     KC_X,     KC_C,  KC_V,  KC_B,  KC_N,    KC_M,  KC_COMM, KC_DOT,   RSFT_T(KC_SLSH), KC_UP,             LT(2, KC_DEL), \
-		KC_LCTL,  KC_LALT,  KC_LGUI,                KC_SPC,                KC_RALT,  MO(1)  , KC_LEFT,         KC_DOWN,           KC_RIGHT),
+		KC_LCTL,  KC_LALT,  KC_LGUI,                KC_SPC,                KC_RGUI,  MO(1)  , KC_LEFT,         KC_DOWN,           KC_RGHT),
 	[_LAYER1] = LAYOUT( /* FN */
 		TO(3),    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_DEL , \
-		KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_CALC,  KC_TRNS,  KC_INS,   KC_TRNS,  KC_PSCR,  KC_SLCK,  KC_PAUS,  RESET  , \
-		KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_HOME,  KC_PGUP,            KC_TRNS, \
-		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_END,   KC_PGDOWN, KC_VOLU,            KC_MUTE, \
-		KC_TRNS,  KC_TRNS,  KC_TRNS,                      TO(4),                        KC_TRNS,  KC_TRNS,  KC_MPRV,  KC_VOLD,            KC_MNXT),
+		KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    LGUI(LSFT(KC_5)),   KC_SLCK,  KC_PAUS,  KC_NO,    KC_NO,   RESET,\
```

Comment:
You can use `XXXXXXX` and `_______` (7 characters) in place of `KC_NO` and `KC_TRNS`, respectively. These make keymaps considerably easier to read :)

Reply:
Oh, I did not know that! Thanks!